### ISSUE TITLE
Default Date format

### ISSUE THREAD

**Issue by wtime** [Laravel-Backpack/CRUD#146]
Hi @tabacitu, in the list page of crud, the date column is not formatted by the configuration format (Config/base.php). I set the format with: 'default_date_format' => 'j F Y'  but in column i see  'Y F j'. On insert or edit mode the format is correct.

**Comment by tabacitu** [comment]
Hi @wtime ,

Hmm... What CRUD version are you using? You can find that out in vendor/backpack/crud/CHANGELOG.md 

Cheers!

**Comment by wtime** [comment]
[3.1.20] - 2016-09-26 thank you

**Comment by tabacitu** [comment]
Wow. 

1) Doesn't your ```vendor/backpack/crud/src/resources/views/columns/date.php``` look like this?

```php
{{-- localized date using jenssegers/date --}}
<td data-order="{{ $entry->{$column['name']} }}">
	{{ Date::parse($entry->{$column['name']})->format(config('backpack.base.default_date_format')) }}
</td>
```

2) Do you happen to have a published view inside your project that overwrites this package file, in ```resources/views/vendor/backpack/columns/date.blade.php```?

**Comment by wtime** [comment]
1) It's the same
2) no, it's the default template, no editing or overwriting
If  i try to change it, nothing is changing, very strange!

**Comment by wtime** [comment]
no news?

**Comment by OwenMelbz** [comment]
@wtime are you setting the column type in your controller?

e.g

```php
$this->crud->addColumn([
    'type' => 'date',
    'name' => 'created_at',
    'label' => 'Created on'
]);
```

**Comment by wtime** [comment]
yes

**Comment by OwenMelbz** [comment]
@wtime I noticed you said that when you edited the template, nothing happened? If you literally delete the template file, does it still work?

if it still works then I think there must be a published version somewhere that is over-writing it

**Comment by wtime** [comment]
I have found the problem! I'm setting the column type with:

$this->crud->setColumns(['firstname','lastname','date']);

Have I to set them with a single addColumn for each field?

**Comment by tabacitu** [comment]
Hi @wtime . Yes, please do.

Personally, I don't use ```$this->crud->setFromDb();``` at all. I follow [this fast workflow](https://laravel-backpack.readme.io/docs/crud-example#doing-it-all-very-fast), then delete setFromDb() and manually type all the columns and fields I need with ```$this->crud->addField();``` and ```$this->crud->addColumn();```. That's because I know setFromDb() isn't very good right now - it does not figure out by itself what the field/column types should be, so I prefer to just add them myself.

However, using ```$this->crud->setFromDb();``` is fine too, whatever you prefer.

Cheers!

**Comment by OwenMelbz** [comment]
@wtime just having a tidy up, closing the issue :) if you want, feel free to re-open it if you need anything else!

**Comment by Jaspur** [comment]
It's still not working. Getting the Y-m-d H:i format while I configured in the config as 'j F Y'

**Comment by sinallcom** [comment]
Its still not working now!

**Comment by tabacitu** [comment]
Hi @sinallcom  - I guess it’s not exactly clear from the conversation above. 

TLDR:
- **in order to have the date shown in the configured format, you have to use the ```date``` or ```datetime``` column types**;
- ```$this->crud->setFromDb();``` will NOT use the proper column type; it will just use ```text```;
- the shorthand (```$this->crud->addField(‘date');``` will NOT use the proper column type; it will just use ```text```;
- the correct usage would be:
```php
$this->crud->addField([
   'name' => "name", // The db column name
   'label' => "Tag Name", // Table column heading
   'type' => "date",
    // 'format' => 'l j F Y', // use something else than the base.default_date_format config value
]);
```

Hope it helps.
Cheers!
